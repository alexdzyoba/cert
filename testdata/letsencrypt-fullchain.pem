-----BEGIN CERTIFICATE-----
MIIFVjCCBD6gAwIBAgISA0ZrZQFjC5n0ywvOvj4QULtyMA0GCSqGSIb3DQEBCwUA
MEoxCzAJBgNVBAYTAlVTMRYwFAYDVQQKEw1MZXQncyBFbmNyeXB0MSMwIQYDVQQD
ExpMZXQncyBFbmNyeXB0IEF1dGhvcml0eSBYMzAeFw0xOTA5MTQxOTIxMzBaFw0x
OTEyMTMxOTIxMzBaMBoxGDAWBgNVBAMTD2FsZXguZHp5b2JhLmNvbTCCASIwDQYJ
KoZIhvcNAQEBBQADggEPADCCAQoCggEBAK5hh5LAnAFB1/TEz+HxzIG3QXKbzOSq
O6+0IlXB1EnLUUXK47KupB951FqObufjpBIG3JNE7xXT7tmkwwovZGy8f7cekZe/
F+MTwW2b0Kxq1zYrIe9gINOKdfC1dPGKTDyJfz2q+AxuYlCSRbXlb39SfATEMiVG
PWyu0Y0fSRWdcmJxm35cbPNZgfB4fPXJhYhMpPUBvQXr38V97qHufrC9nLrgxia7
2Oo9VJREL6I9eVzMkEIe9haqZbRWeByJU5LVhahQcB0+NWVj5gF5P8OmUme761W0
xM27T7WaM/5zZ5rGL46LhgNLRwVa1PizOOjvsxgwQKV5a937rrHsUSkCAwEAAaOC
AmQwggJgMA4GA1UdDwEB/wQEAwIFoDAdBgNVHSUEFjAUBggrBgEFBQcDAQYIKwYB
BQUHAwIwDAYDVR0TAQH/BAIwADAdBgNVHQ4EFgQUdO+PXNXNJssn8I5Ob4713kWz
CCUwHwYDVR0jBBgwFoAUqEpqYwR93brm0Tm3pkVl7/Oo7KEwbwYIKwYBBQUHAQEE
YzBhMC4GCCsGAQUFBzABhiJodHRwOi8vb2NzcC5pbnQteDMubGV0c2VuY3J5cHQu
b3JnMC8GCCsGAQUFBzAChiNodHRwOi8vY2VydC5pbnQteDMubGV0c2VuY3J5cHQu
b3JnLzAaBgNVHREEEzARgg9hbGV4LmR6eW9iYS5jb20wTAYDVR0gBEUwQzAIBgZn
gQwBAgEwNwYLKwYBBAGC3xMBAQEwKDAmBggrBgEFBQcCARYaaHR0cDovL2Nwcy5s
ZXRzZW5jcnlwdC5vcmcwggEEBgorBgEEAdZ5AgQCBIH1BIHyAPAAdQDiaUuuJujp
QAnohhu2O4PUPuf+dIj7pI8okwGd3fHb/gAAAW0xcBEEAAAEAwBGMEQCIBt+p9OH
SokfOHOOISHOpnkmiCemFgxb2ctOvoiLCTiDAiA5ZRwbbFjY2pHuusFFD2Hk7Vyr
qfJJhadF9K9qO0kbjQB3AGPy283oO8wszwtyhCdXazOkjWF3j711pjixx2hUS9iN
AAABbTFwDwgAAAQDAEgwRgIhAJrgITZK0azNmnPxajXu32ZMSXXNom2sByUzoSTj
oNwFAiEA0C9Gh5lJi+T+Z/TkQCsp0WprPgIM8uLMn7vTs4djPUcwDQYJKoZIhvcN
AQELBQADggEBABgBCyTVSwO9AcNFH8217Ye+ZAv1TInkP4dbrDqd8wwzhXbV6nH2
eMBDwd38DkHJQUOzrjTgzAFGjpH0BtadcdlnxGBcWc4O+avEy/lLd4EFZC9sjsjK
D3XO6e4/nIKm3NVNxf5lTpFWg0VJHEJKopnMluDjFIcXFvCrPWTLhzBgfXVEUwPC
zLr3Ur5CCyrhWmVgnDAzE9zHalWzBJMyfYrc/sIRGprqiebycPxlRl4dwy/k18fr
s4GUrUrh19rwBMtLAl9qluajfcOwoLtT3QBjj3pzpuT8+yvA+sli++XKQeAbUf3Z
F3B6ZY5cYTXlPH2DFgP3TWNGbz5pXuy7G80=
-----END CERTIFICATE-----
-----BEGIN CERTIFICATE-----
MIIEkjCCA3qgAwIBAgIQCgFBQgAAAVOFc2oLheynCDANBgkqhkiG9w0BAQsFADA/
MSQwIgYDVQQKExtEaWdpdGFsIFNpZ25hdHVyZSBUcnVzdCBDby4xFzAVBgNVBAMT
DkRTVCBSb290IENBIFgzMB4XDTE2MDMxNzE2NDA0NloXDTIxMDMxNzE2NDA0Nlow
SjELMAkGA1UEBhMCVVMxFjAUBgNVBAoTDUxldCdzIEVuY3J5cHQxIzAhBgNVBAMT
GkxldCdzIEVuY3J5cHQgQXV0aG9yaXR5IFgzMIIBIjANBgkqhkiG9w0BAQEFAAOC
AQ8AMIIBCgKCAQEAnNMM8FrlLke3cl03g7NoYzDq1zUmGSXhvb418XCSL7e4S0EF
q6meNQhY7LEqxGiHC6PjdeTm86dicbp5gWAf15Gan/PQeGdxyGkOlZHP/uaZ6WA8
SMx+yk13EiSdRxta67nsHjcAHJyse6cF6s5K671B5TaYucv9bTyWaN8jKkKQDIZ0
Z8h/pZq4UmEUEz9l6YKHy9v6Dlb2honzhT+Xhq+w3Brvaw2VFn3EK6BlspkENnWA
a6xK8xuQSXgvopZPKiAlKQTGdMDQMc2PMTiVFrqoM7hD8bEfwzB/onkxEz0tNvjj
/PIzark5McWvxI0NHWQWM6r6hCm21AvA2H3DkwIDAQABo4IBfTCCAXkwEgYDVR0T
AQH/BAgwBgEB/wIBADAOBgNVHQ8BAf8EBAMCAYYwfwYIKwYBBQUHAQEEczBxMDIG
CCsGAQUFBzABhiZodHRwOi8vaXNyZy50cnVzdGlkLm9jc3AuaWRlbnRydXN0LmNv
bTA7BggrBgEFBQcwAoYvaHR0cDovL2FwcHMuaWRlbnRydXN0LmNvbS9yb290cy9k
c3Ryb290Y2F4My5wN2MwHwYDVR0jBBgwFoAUxKexpHsscfrb4UuQdf/EFWCFiRAw
VAYDVR0gBE0wSzAIBgZngQwBAgEwPwYLKwYBBAGC3xMBAQEwMDAuBggrBgEFBQcC
ARYiaHR0cDovL2Nwcy5yb290LXgxLmxldHNlbmNyeXB0Lm9yZzA8BgNVHR8ENTAz
MDGgL6AthitodHRwOi8vY3JsLmlkZW50cnVzdC5jb20vRFNUUk9PVENBWDNDUkwu
Y3JsMB0GA1UdDgQWBBSoSmpjBH3duubRObemRWXv86jsoTANBgkqhkiG9w0BAQsF
AAOCAQEA3TPXEfNjWDjdGBX7CVW+dla5cEilaUcne8IkCJLxWh9KEik3JHRRHGJo
uM2VcGfl96S8TihRzZvoroed6ti6WqEBmtzw3Wodatg+VyOeph4EYpr/1wXKtx8/
wApIvJSwtmVi4MFU5aMqrSDE6ea73Mj2tcMyo5jMd6jmeWUHK8so/joWUoHOUgwu
X4Po1QYz+3dszkDqMp4fklxBwXRsW10KXzPMTZ+sOPAveyxindmjkW8lGy+QsRlG
PfZ+G6Z6h7mjem0Y+iWlkYcV4PIWL1iwBi8saCbGS5jN2p8M+X+Q7UNKEkROb3N6
KOqkqm57TH2H3eDJAkSnh6/DNFu0Qg==
-----END CERTIFICATE-----
