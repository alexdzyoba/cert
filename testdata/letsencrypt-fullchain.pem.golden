[0] Certificate:
serial number: 285299375325407718547624347336838676003698
subject: CN=alex.dzyoba.com
issuer: CN=Let's Encrypt Authority X3,O=Let's Encrypt,C=US
isCA: false
root: false
valid: from '2019-09-14 19:21:30' to '2019-12-13 19:21:30'

[1] Certificate:
serial number: 13298795840390663119752826058995181320
subject: CN=Let's Encrypt Authority X3,O=Let's Encrypt,C=US
issuer: CN=DST Root CA X3,O=Digital Signature Trust Co.
isCA: true
root: false
valid: from '2016-03-17 16:40:46' to '2021-03-17 16:40:46'

